drivers/pwm/pwm_stm32.c

```diff
@@ -103,6 +103,14 @@ static const uint32_t ch2ll[TIMER_MAX_CH] = {
 #endif
 };
 
+static const uint32_t ch2ll_n[TIMER_MAX_CH] = {
+	LL_TIM_CHANNEL_CH1N, LL_TIM_CHANNEL_CH2N,
+	LL_TIM_CHANNEL_CH3N, LL_TIM_CHANNEL_CH4,
```

Comment:
Depending on STM32 series, CH4 can have a complementary channel,like on STM32G4, U5, ...
or not, like STM32L4, ...
Maybe macro IS_TIM_COMPLEMENTARY_CHANNELS  could be used.